src/core/selectors.ts

```diff
@@ -1,24 +1,62 @@
 import { Request } from 'groupby-api';
+import { QueryTimeAutocompleteConfig, QueryTimeProductSearchConfig } from 'sayt';
+import Autocomplete from './adapters/autocomplete';
+import Configuration from './adapters/configuration';
 import { MAX_RECORDS } from './adapters/search';
+import AppConfig from './configuration';
 import Store from './store';
 
 namespace Selectors {
 
-  export const searchRequest = (state: Store.State): Request => {
+  export const searchRequest = (state: Store.State, config: AppConfig): Request => {
     const sort = Selectors.sort(state);
     const pageSize = Selectors.pageSize(state);
     const skip = Selectors.skip(state, pageSize);
-
-    return <any>{
+    const request: Partial<Request> = {
       pageSize: Math.min(pageSize, MAX_RECORDS - skip),
       area: Selectors.area(state),
       fields: Selectors.fields(state),
       query: Selectors.query(state),
       collection: Selectors.collection(state),
       refinements: Selectors.selectedRefinements(state),
-      sort: sort && Selectors.requestSort(sort),
       skip
     };
+
+    const language = Configuration.extractLanguage(config);
+    if (language) {
+      request.language = language;
+    }
+    if (sort) {
+      request.sort = <any>Selectors.requestSort(sort);
+    }
+
+    return <Request>{
+      ...config.search.defaults,
+      ...request,
+      ...config.search.overrides,
```

Comment:
I want to make them mandatory at some point, because it makes editing in an IDE easier, but I'm fine with them being around messily at the moment